QualityControl/public/Model.js

```diff
@@ -261,7 +261,7 @@ export default class Model extends Observable {
       case 'about':
         this.page = 'about';
         setBrowserTabTitle('QCG-About');
-        this.frameworkInfo.getFrameworkInfo();
+        this.aboutViewModel.getAboutViewModel();
```

Comment:
Hi @mariscalromeroalejandro. What is the reason for renaming this method to suggest that a model is retrieved?